.travis.yml

```diff
@@ -4,13 +4,17 @@ git:
 env:
   global:
     - ATOM_ACCESS_TOKEN=da809a6077bb1b0aa7c5623f7b2d5f1fec2faae4
-    
-  matrix:
     - NODE_VERSION=0.12
 
+matrix:
+  include:
+    - os: osx
+      env: ATOM_SPECS_TASK=core
+    - os: osx
+      env: ATOM_SPECS_TASK=packages
+
 os:
   - linux
-  - osx
```

Comment:
Great! :sparkles: Do you think we can remove `- linux` from here and move it to the `matrix` section for the sake of consistency?

Then, if at some point we’ll manage to run specs on Linux as well we can simply ditch `matrix: include`, set the environment variables in the global matrix and reintroduce the `os` section.